policy/lib/tekton/task.rego

```diff
@@ -114,6 +114,16 @@ build_task(attestation) := task if {
 	count(trim_space(image_digest)) > 0
 }
 
+git_clone_task(attestation) := task if {
```

Comment:
I know we're all in on git, but we could do some cheap future proofing for when we need to support something else, say:
```suggestion
scm_fetch_task(attestation) := task if {
```
And for other mentions of git...

Reply:
I think `git` is right in this context since that's all that we support. For example, Chains the `git+` prefix unconditionally!